cf-agent/verify_files.c

```diff
@@ -585,6 +585,15 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
     case PROMISE_RESULT_CHANGE:
         cfPS(ctx, LOG_LEVEL_INFO, result, pp, &a,
              "files promise '%s' repaired", pp->promiser);
+
+        if (ChrootChanges() && !a.haverename)
```

Comment:
Is it possible to do rename AND something else with the same promise? How will this work?

Reply:
I don't think that's possible. That's why recording renames is handled in the rename code and those cases are skipped here.